docs/Token_Assumptions.md

```diff
@@ -0,0 +1,21 @@
+_Attestation Token use cases & assumptions_
+The following captures what is assumed about the nature of the attestation tokens received for verification.
+
+A token is presented as a single 'blob' (byte stream).
+
+Tokens are presented individually. If a batch mode submission is required, that is the responsibility of an API tier.
+
+Encoding in transit is the responsibility of the API tier, the verification pipeline works on a byte stream.
+
+The verification pipeline will receive some context information which will indicate or help to deduce the nature of the token and hence which plugins will be needed in the verification pipeline. In a dedicated deployment this may be hardwired, in an environment supporting multiple token types, this is likely to come from the API tier.
+
+A token may have an encrypted body. Assuming the correct decryption key is available, the body will be decrypted and then passed into the pipeline in the same manner as an unencrypted token.
+
+A presented token blob may actually contain multiple tokens, each of which will be handled e.g. deserialised and have signing checked, independently such that an aggregation of all evidence contained is extracted and presented for appraisal
+
+Multiple token models include both concatenation (e.g a chain of certs packaged serially) or embedded (e.g. the value of a claim within one token may be a serialised independent token)
+
```

Comment:
I don't understand this bit of the sentence: "i.e. the verifier cannot distinguish between a bad value in a claim and a 'missing' value."

Reply:
I was trying to underline the nature of the workflow: we assume that there isn't a 'live' connection between the use of the attested environment and the supply chain values that match that environment. Thus if the verifier does not find a RV match, there could be two explanations: bad value or missing RV, it's not within the verifier scope to distinguish the two.

Reply:
OK, thanks.  Now I get the intent.  I'm worried that making this explicit in this context may baffle the simple minds (like myself!) without adding real substance to what we are trying to say?

Reply:
Don't want to confuse so will remove this sentence

Reply:
removed to calm cerebral squalls